data/transit/route/ferry.json

```diff
@@ -306,8 +306,10 @@
         "include": ["au-qld.geojson"]
       },
       "tags": {
-        "network": "RiverCity Ferries",
-        "network:wikidata": "Q97194713",
+        "network": "Translink",
```

Comment:
Maybe you've mixed network and operator? Because Translink is a company that actual management ferry run, and River City Ferries is network, like "Beijing Bus", "Tokyo Metro", "Londun Bus".

Reply:
The operator of the ferry services are RiverCity Ferries however all the services they operate are part of the Translink network. The fares/zones that the ferry services operate in are determined by Translink, not RiverCity Ferries. It is the same case with buses and trains where they are operated by their respected operators however are part of the Translink fare network.

Reply:
It stated on the Translink wikipedia page the following:
"Translink works with Brisbane Airtrain, Transport for Brisbane, RiverCity Ferries, Queensland Rail and other operators to provide services." As a person who is from Brisbane (where Translink is), Translink is the network.